pkgs/applications/networking/cluster/k3s/docs/USAGE.md

```diff
@@ -21,7 +21,6 @@
 ```
 
 Once the above changes are active, you can access your cluster through `sudo k3s kubectl` (e.g. `sudo k3s kubectl cluster-info`) or by using the generated kubeconfig file in `/etc/rancher/k3s/k3s.yaml`.
-Multi-node setup
```

Comment:
This is just me also fixing a typo I saw in this same commit. Not relevant to the k3s functionality, just a formatting thing.